### PR TITLE
Added strokeWidth to RefreshIndicator

### LINKED ISSUE #42570: RefreshIndicator missing stroke width?
`RefreshIndicator` doesn't have `strokeWidth` parameter, that `RefreshProgressIndicator`, which is used inside it, does

### PR DESCRIPTION
**Pull request by keshava19**
## Description

Added `strokeWidth` property to the `RefreshIndicator`.

## Related Issues

Fixes #42570

## Tests

I added the following tests:

`RefreshIndicator - default strokeWidth`
`RefreshIndicator - custom strokeWidth`

in file `refresh_indicator_test.dart`

## Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

**Review by darielopper — APPROVED** [review]

File: packages/flutter/test/material/refresh_indicator_test.dart [hunk 1]
```diff
@@ -423,4 +423,57 @@ void main() {
 
     debugDefaultTargetPlatformOverride = null;
   });
+
+  testWidgets('RefreshIndicator - default strokeWidth', (WidgetTester tester) async {
+    refreshCalled = true;
```
Comment (Piinks) [inline packages/flutter/test/material/refresh_indicator_test.dart]: What is this for?

File: packages/flutter/test/material/refresh_indicator_test.dart [hunk 2]
```diff
@@ -423,4 +423,57 @@ void main() {
...
+        ),
+      ),
+    );
+
+    final RefreshIndicator refreshIndicator = tester.widget(find.byKey(refreshIndicatorKey));
+    expect(refreshIndicator.strokeWidth, defaultStrokeWidth);
+  });
+
+  testWidgets('RefreshIndicator - custom strokeWidth', (WidgetTester tester) async {
+    refreshCalled = true;
```
Comment (Piinks) [inline packages/flutter/test/material/refresh_indicator_test.dart]: Here too.

**Review by Piinks — COMMENTED** [review]
Hi @keshava19, thank you for the contribution. Can you update your branch to resolve the merge conflicts here?

**Comment by Piinks** [comment]
Thank you for the contribution. Since we haven't seen a reply, we're going to reluctantly close this pull request for now, but if you get time to address the comments and resolve the merge conflicts, we'd be more than happy to take a look at a new patch. If you do send a new patch, it would be very useful to reference this one in the description in order to help reviewers see the previous context.

**Pull request closed without merging**